web/src/components/GuidanceListPage/GuidanceListPage.tsx

```diff
@@ -111,10 +111,15 @@ export const getGuidanceListPage =
 			} = results as SearchResultsSuccess;
 
 		useEffect(() => {
-			setAnnouncement(
-				`Showing ${firstResult} to ${lastResult} of ${resultCount}`
-			);
-		}, [firstResult, lastResult, resultCount]);
+			if (resultCount === 0) {
+				setAnnouncement("No results found");
+			} else {
+				const sortOrder = s == "Title" ? "title" : "date";
```

Comment:
I'm not sure this works for all tabs unfortunately. Indev/proposed are sorted by title by default if there's no sort order in the URL. So we might need to use `defaultSort` here

Reply:
But worth double checking on all 4 tabs as you can quickly verify that I think